airbyte-server/src/main/java/io/airbyte/server/ConfigDumpExport.java

```diff
@@ -93,7 +94,8 @@ private void exportVersionFile(Path tempFolder) throws IOException {
   }
 
   private void dumpDatabase(Path parentFolder) throws Exception {
-    final Map<String, Stream<JsonNode>> tables = jobPersistence.dump();
+    final Map<String, Stream<JsonNode>> tables = jobPersistence.exportDatabase().entrySet().stream()
+        .collect(Collectors.toMap(e -> e.getKey().name(), Entry::getValue));
```

Comment:
If we dump any table that exists in the database, the migration will break [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-migration/src/main/java/io/airbyte/migrate/Migrate.java#L165), because each extra table will result in its own `ResourceId` for `airbyte_job`, which does not match any input stream from the migration. This concern was previously raised by @ChristopheDuong in https://github.com/airbytehq/airbyte/pull/3980#discussion_r652696946.

@subodh1810, is there any concern to change the `dump` method here to `exportDatabase`? In the future, the database is no longer just for job tables. The config table may exist there as well.